.gitignore

```diff
@@ -48,6 +48,11 @@ terraform.rc
 *.pem
 *.p12
 *.pub
+*.pfx
+*.crt
 
 #Licenses
 *.license
+
+#Private key
+privatekey.key
```

Comment:
What if the private key is named differently?

Reply:
It would push it. I could change it to *.key but not sure if it is a good idea?

Reply:
Ofc it's not ok. That's why I am adding the git ignore, so someone doesn't push his key by mistake.

I would put *.key in git ignore so someone would not push his key accidentally